tests/test_computational_resources.py

```diff
@@ -511,7 +511,7 @@ def test_resource_setup_widget_default():
     # and the computer/code setup widget will be updated accordingly.
     w.comp_resources_database.domain_selector.value = "daint.cscs.ch"
     w.comp_resources_database.computer_selector.value = "mc"
-    w.comp_resources_database.code_selector.value = "QuantumESPRESSO-7.2"
```

Comment:
This should fail because there is no `QuantumESPRESSO-7.2` in resource registry, we remove it in https://github.com/aiidateam/aiida-resource-registry/commit/a6b1dc567fa9114afd99ebe7c3404117c81120a1

Reply:
Can you make it to a separate PR? We're run into this already elsewhere and should merge asap independent of this PR.

Reply:
Ah, that make sense, simply I didn't realize it because no more test running in AWB. PR opened in https://github.com/aiidalab/aiidalab-widgets-base/pull/544